map_test.go

```diff
@@ -1033,6 +1040,93 @@ func TestPerCPUMarshaling(t *testing.T) {
 	}
 }
 
+func TestCgroupPerCPUStorageMarshaling(t *testing.T) {
+	numCPU, err := internal.PossibleCPUs()
+	if err != nil {
+		t.Fatal(err)
+	}
+	if numCPU < 2 {
+		t.Skip("Test requires at least two CPUs")
+	}
+	testutils.SkipOnOldKernel(t, "5.9", "per-CPU CGoup storage with write from user space support")
+
+	cgroup := testutils.CreateCgroup(t)
+
+	arr, err := NewMap(&MapSpec{
+		Type:      PerCPUCGroupStorage,
+		KeySize:   16,
```

Comment:
If the map key is a `bpfCgroupStorageKey`, couldn't `unsafe.Sizeof()` be used instead? Then there's no magic constant.

Reply:
Sure.